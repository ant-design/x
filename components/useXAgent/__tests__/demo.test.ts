import demoTest from '../../../tests/shared/demoTest';

demoTest('useXAgent');
