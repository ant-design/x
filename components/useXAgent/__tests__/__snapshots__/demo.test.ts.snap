// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders components/useXAgent/demo/custom.tsx correctly 1`] = `
Array [
  <form
    autocomplete="off"
    class="ant-form ant-form-vertical css-var-R0 ant-form-css-var"
  >
    <div
      class="ant-form-item css-var-R0 ant-form-css-var"
    >
      <div
        class="ant-row ant-form-item-row css-var-R0"
      >
        <div
          class="ant-col ant-form-item-label css-var-R0"
        >
          <label
            class=""
            for="question"
            title="Question"
          >
            Question
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control css-var-R0"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input ant-input-outlined css-var-R0 ant-input-css-var"
                id="question"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <button
      class="ant-btn css-var-R0 ant-btn-primary"
      type="submit"
    >
      <span>
        submit
      </span>
    </button>
  </form>,
  <div
    class="ant-divider css-var-R0 ant-divider-horizontal"
    role="separator"
  />,
  <article
    class="ant-typography css-var-R0"
  >
    <ul />
  </article>,
]
`;

exports[`renders components/useXAgent/demo/preset.tsx correctly 1`] = `
<div>
  Wait for XFetch & XStream
</div>
`;
